src/app.js

```diff
@@ -43,6 +44,20 @@ export default class App extends React.Component {
     }
   }
 
+  renderModeView() {
```

Comment:
What is the difference between `renderModeView` and `renderView`? Maybe `renderView` could be renamed to something else if it's always rendering the same thing? If you have to, just split out a new `renderSomething` method.